### PR TITLE
(VANAGON-64) Update changelog for 0.13.0

### PR DESCRIPTION
**Pull request by demophoon**
Updates changelog for 0.13.0 release of vanagon

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.13.0] - released on 2017-07-12
+### Added
+ - A metadata file is generated when producing artifacts which contains metadata about components and various other information about a built artifact.
```
Comment (mckern) [inline CHANGELOG.md]: Suggestion: maybe document the broad outline of what can be expected to be in the metadata file for this release?

Comment (demophoon) [inline CHANGELOG.md]: Wasn't sure if that would be too specific for a changelog or not, but I can definitely go into more details here.

Comment (mckern) [inline CHANGELOG.md]: The rule I usually used was "if there's no documentation written for it in the README, document it in the changelog"

Comment (demophoon) [inline CHANGELOG.md]: @mckern How does this look to you?

Comment (mckern) [inline CHANGELOG.md]: It looks like you're a responsible and professional developer.

Comment (demophoon) [inline CHANGELOG.md]: If we redefine the terms 'responsible', 'professional', and 'developer' 😄

**Pull request merged**